### PR TITLE
Update election code, client and godeps

### PR DESCRIPTION
**Pull request by aledbf**

### CODE REVIEW

**Comment by bprashanth** [comment]
@mikedanese review? Not sure who is point on this

**Comment by mikedanese** [comment]
Sure, I can take a look.

File: election/Makefile [hunk 1]
```diff
@@ -1,17 +1,22 @@
 all: push
 
 # 0.0 shouldn't clobber any released builds
-TAG = 0.0
+TAG = 0.5
```
Comment (mikedanese) [inline election/Makefile]: Comment above is irrelevent if we want to make this change. How about
```Makefile
# 0.0 shouldn't clobber any released builds
# current latest is 0.5
TAG ?= 0.0
```
So tag will needed to be provided explicitly to deploy new version to gcr.io.

File: election/client/nodejs/Dockerfile [hunk 2]
```diff
@@ -1,3 +1,5 @@
-FROM node:0.12
+FROM node:6.3-slim
```
Comment (mikedanese) [inline election/client/nodejs/Dockerfile]: Major bump :). Is this basically latest? (disclosure: I know nothing about node or javascript).

**Comment by mikedanese** [comment]
Changes looks sensible. The diffstat (+824 −18,815) seems wrong... but If tests pass, let's merge. Thanks for keeping up to date. I'll do a push to gcr.io after merging.

File: election/client/nodejs/Dockerfile (hunk 2 above)
Comment (aledbf) [inline election/client/nodejs/Dockerfile]: > Is this basically latest?

Yes, 6.3 is the latest stable version (check [this](https://nodejs.org/en/download/releases/) to see how old is 0.12 )

File: election/Makefile (hunk 1 above)
Comment (aledbf) [inline election/Makefile]: done

**Comment by mikedanese** [comment]
pushed

**Pull request merged**